### ISSUE TITLE
DBML export option

### ISSUE THREAD

**Issue by boxysean** [ScalefreeCOM/turbovault4dbt#26]
I was speaking with a team about TurboVault, and they identified that dbdocs.io would be an additional piece of software for them to manage. My suggestion: allow for a DBML export via the TurboVault UI, in addition to the dbdocs.io integration, so that teams can choose how to visualize their DBML file based on their corporate requirements.

**Comment by universe-designer** [comment]
I agree. In my project environment cloud services are not allowed. A common format would be better.
